ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -203,13 +203,16 @@ object CommandsValidator {
   }
 
   def effectiveSubmitters(commands: ProtoCommands): Submitters[String] = {
-    val effectiveActAs =
-      if (commands.party.isEmpty)
-        commands.actAs.toSet
-      else
-        commands.actAs.toSet + commands.party
-    val effectiveReadAs = commands.readAs.toSet -- effectiveActAs
-    Submitters(effectiveActAs, effectiveReadAs)
+    val actAs = effectiveActAs(commands)
+    val readAs = commands.readAs.toSet -- actAs
+    Submitters(actAs, readAs)
+  }
+
+  def effectiveActAs(commands: ProtoCommands): Set[String] = {
```

Comment:
We can get rid of these braces. Someone may also want to add some for the `if` expression ;)

Reply:
I like to skip the braces for simple `if` expressions.

Reply:
me 2 :)